index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>main</title>
+</head>
+<body>
+    <div class="header1"><h2>We're open and available for takeaway & delivery. Order Now</h2></div>

```

Comment:
You need to learn `BEM` https://en.bem.info/methodology/quick-start/
It helps you and me to work on the feature...